deeplake/core/storage/memory.py

```diff
@@ -124,3 +125,8 @@ def __setstate__(self, state: str):
 
     def get_object_size(self, key: str) -> int:
         return _get_nbytes(self[key])
+
+    def subdir(self, path: str, read_only: bool = False):
```

Comment:
@farizrahman4u What does this section do?

Reply:
This method was missing in memeoryprovider, which I found out while testing view stuff with memory ds.